### PR TITLE
BACKEND: API Prefix to all backend routes; Register renamed to Signup, bug fixes with mappers

### PR DESCRIPTION
**Pull request by alxedelweiss**
Please navigate to the 'fix-transaction' branch and test the backend.

You should be able to seed the file and run app.py. On postman, you should be able to test the '/api/signup' route with a 201 successful code. If a field is missing, we are notified by the backend and the error is returned to the user.

### CODE REVIEW

**Review by LandonCramer — COMMENTED** [review]
Looks good

**Comment by alxedelweiss** [comment]
Need an approval before I can merge it!

**Review by LandonCramer — APPROVED** [review]

**Pull request merged**
